notebooks/regression/exoplanets.md

```diff
@@ -58,7 +59,7 @@ warnings.filterwarnings("ignore")
 ## 1. Data Loading
 
 
-Let's start by loading the exoplanet dataset and show the main information.
+Let's start by loading the `exoplanets` dataset and seeing the main information.
```

Comment:
seeing -> looking to (sorry for my mistake)

Reply:
Changed to "looking at".

Reply:
OK